dbms/src/Columns/ColumnConst.cpp

```diff
@@ -103,4 +106,18 @@ void ColumnConst::getPermutation(bool /*reverse*/, size_t /*limit*/, int /*nan_d
         res[i] = i;
 }
 
+void ColumnConst::updateWeakHash32(WeakHash32 & hash) const
```

Comment:
Maybe don't update hash at all?

Reply:
I think we need it. 
Because data could be represented as const column, but different in different chunks.
And also hash will be unreasonably different after materializing.